diff --git a/node_modules/@stripe/stripe-react-native/android/src/main/java/com/reactnativestripesdk/utils/PostalCodeUtilities.kt b/node_modules/@stripe/stripe-react-native/android/src/main/java/com/reactnativestripesdk/utils/PostalCodeUtilities.kt
index 3333429..6586518 100644
--- a/node_modules/@stripe/stripe-react-native/android/src/main/java/com/reactnativestripesdk/utils/PostalCodeUtilities.kt
+++ b/node_modules/@stripe/stripe-react-native/android/src/main/java/com/reactnativestripesdk/utils/PostalCodeUtilities.kt
@@ -2,8 +2,8 @@ package com.reactnativestripesdk.utils
 
 class PostalCodeUtilities {
   companion object {
-    internal fun isValidGlobalPostalCodeCharacter(c: Char): Boolean = Character.isLetterOrDigit(c) || c.isWhitespace() || c == '-'
+    internal fun isValidGlobalPostalCodeCharacter(c: Char): Boolean = Character.isLetterOrDigit(c) || Character.isWhitespace(c) || c == '-'
 
-    internal fun isValidUsPostalCodeCharacter(c: Char): Boolean = Character.isDigit(c) || c.isWhitespace() || c == '-'
+    internal fun isValidUsPostalCodeCharacter(c: Char): Boolean = Character.isDigit(c) || Character.isWhitespace(c) || c == '-'
   }
 }
